src/i18n.js

```diff
@@ -0,0 +1,21 @@
+import i18n from "i18next";
+import Backend from "i18next-http-backend";
+import LanguageDetector from "i18next-browser-languagedetector";
+import { initReactI18next } from "react-i18next";
+
+i18n
+  .use(Backend)
```

Comment:
It is probably easier and has more sense to load translations directly in from json files than doing a separate http request. Keep in mind we have a really small set of translations